src/service/parallax-impl.js

```diff
@@ -35,12 +34,6 @@ export class ParallaxService {
    * @param {!./ampdoc-impl.AmpDoc} ampdoc
    */
   constructor(ampdoc) {
-    /** @private @const {!Observable} */
-    this.parallaxObservable_ = new Observable();
```

Comment:
I noticed you removed this listener from the `ParallaxService`, but didn't add something similar to the `ParallaxElement` class. You should still provide a way to detect scroll events to use in [`extensions/amp-fx-parallax/0.1/test/test-amp-fx-parallax.js`](https://github.com/aghassemi/amphtml/blob/869dd1ca1e563769f7c6701d699867d00a8819ab/extensions/amp-fx-parallax/0.1/test/test-amp-fx-parallax.js). I think the current PR will break that test.